### PR TITLE
opengov precompile and pallet translations

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

This PR contains translations from the changes made in: https://github.com/PureStake/moonbeam-docs/pull/546

### CODE REVIEW

File: .snippets/text/precompiles/governance/submit-preimage.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+In this section, you'll get the preimage hash and the encoded proposal data for a proposal. To get the preimage hash, you'll first need to navigate to the **Preimage** page of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network%2Fpublic-ws#){target=_blank}:
+
+ 1. Navigate to the [**Governance** tab](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network%2Fpublic-ws#/democracy){target=_blank}
+ 2. Select **Preimages** from the dropdown
+ 3. From the **Preimages** page, click on **+ Add preimage**
```
Comment (eshaben) [inline .snippets/text/precompiles/governance/submit-preimage.md:5]: This section still needs to be translated, please!

File: .snippets/text/precompiles/governance/submit-preimage.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+![Add a new preimage](/images/builders/pallets-precompiles/precompiles/democracy/democracy-4.png)
+
+Then take the following steps:
```
Comment (eshaben) [inline .snippets/text/precompiles/governance/submit-preimage.md:9]: These lines still needs to be translated, please!

File: builders/pallets-precompiles/pallets/.pages
Comment (eshaben) [inline builders/pallets-precompiles/pallets/.pages:1]: The titles still needs to be translated (if applicable), please!

File: builders/pallets-precompiles/precompiles/.pages
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/.pages:1]: The titles still needs to be translated (if applicable), please!

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 3]
```diff
@@ -12,25 +12,19 @@ keywords: 标准合约, 以太坊, moonbeam, 预编译, 智能合约, 民主
 
 作为波卡（Polkadot）的平行链和去中心化网络，Moonbeam具有原生链上治理功能，使利益相关者能够参与网络的发展方向。要了解有关治理的更多信息，例如相关术语、原则、机制等的概述，请参阅[Moonbeam治理](/learn/features/governance){target=_blank}页面。
 
+With the rollout of OpenGov (originally referred to as Governance v2), several modifications have been introduced to the governance process. **OpenGov has launched on Moonriver, and once it has been rigorously tested, a proposal will be made for it to be launched on Moonbeam**. Until then, Moonbeam still uses Goverance v1. **The Democracy Precompile is for Governance v1, as such, this guide is for Moonbeam only.** If you're looking to interact with governance features on Moonriver, you should take a look at the OpenGov-related precompiles: [Preimage Precompile](/builders/pallets-precompiles/precompiles/preimage){target=_blank}, [Referenda Precompile](/builders/pallets-precompiles/precompiles/referenda){target=_blank}, and [Conviction Voting Precompile](/builders/pallets-precompiles/precompiles/conviction-voting){target=_blank}.
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/democracy.md:15]: This section is new and needs to be translated still, please!

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 4]
```diff
@@ -12,25 +12,19 @@ keywords: 标准合约, 以太坊, moonbeam, 预编译, 智能合约, 民主
...
+
 Moonbeam的链上治理系统得益于[Substrate民主pallet](https://docs.rs/pallet-democracy/latest/pallet_democracy/){target=_blank}。民主pallet使用Rust语言实现，无法直接从Moonbeam的以太坊API交互。然而，民主预编译让您能够直接通过Solidity接口直接访问Substrate民主pallet的治理功能。除此之外，这将能够大幅度改进终端用户的操作体验。举例而言，Token持有者将能够直接使用MetaMask进行公投，无需将账户导入Polkadot.js App后使用复杂的界面进行操作。
 
+The Democracy Precompile is currently available in OpenGov, which is available on Moonriver and Moonbase Alpha only. If you're looking for similar functionality for Moonbeam, which is still on Governance v1, you can refer to the [Democracy Precompile](/builders/pallets-precompiles/precompiles/democracy){target=_blank} documentation.
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/democracy.md:19]: This section is new and needs to be translated still, please!

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 5]
```diff
@@ -113,25 +107,18 @@ Before diving into the interface, it's best if you're familiar with [how to prop
...
-
- 1. 选取一个账户（任何账户皆可，因为您不需要提交任何交易）
- 2. 选取您希望交互的pallet以及可调度的函数（或是动作）以进行提案。您选取的动作将会决定您随后的操作步骤。在此例子中，此为**system** pallet和**remark**函数
- 3. 输入remark函数的内容，确保其为独特的。重复的提案如“Hello World！”将不会被接受
- 4. 复制原像哈希，这代表着此提案。您将会在通过民主预编译提交提案时使用此哈希
- 5. 点击**Submit preimage**按钮，但请不要在下一页签署和确认此交易
-
-![Get the proposal hash](/images/builders/pallets-precompiles/precompiles/democracy/democracy-5.png)
+You can submit a proposal via the `propose` function of the [Democracy Precompile](https://github.com/PureStake/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=_blank} as long as you have the preimage hash of the proposal. But before a proposal can be submitted, you'll first need to submit the preimage by passing in the encoded proposal data to the `notePreimage` function, which now belongs to the [preimage pallet](/builders/pallets-precompiles/pallets/preimage){target=_blank}.
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/democracy.md:110]: This section is new and needs to be translated still, please!

File: builders/pallets-precompiles/pallets/democracy.md
Comment (eshaben) [inline builders/pallets-precompiles/pallets/democracy.md:1]: The changes in this file are from https://github.com/PureStake/moonbeam-docs/pull/512/ - so only some of the things need to be checked and possibly translated

File: builders/pallets-precompiles/pallets/democracy.md [hunk 6]
```diff
@@ -83,8 +109,15 @@ keywords: 民主、substrate、pallet、moonbeam、波卡
 - **instantAllowed**() - 返回一个指示是否发生紧急情况的布林值
 - **launchPeriod**() - 返回公投发布的区块频率
 - **maxProposals**() - 返回公众提案能够在任何时间存在的最大数量
+
+- **maxBlacklisted**() - returns the maximum number of items which can be blacklisted
+- **maxDeposits**() - returns the maximum number of deposits a public proposal may have at any time
```
Comment (eshaben) [inline builders/pallets-precompiles/pallets/democracy.md:114]: These are both totally new and need to be translated, please!

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 7]
```diff
@@ -69,6 +51,23 @@ Moonbeam的链上治理系统得益于[Substrate民主pallet](https://docs.rs/pa
...
+ - **noteImminentPreimage**(*bytes* encodedProposal) —— 为即将到来的提案注册一个原像（Preimage）。此函数需要提案处在调度队列中，同时不需要任何押金。当调用成功，例如：原像（Preimage）尚未被上传且与近期的提案相匹配，不支付任何费用。调用民主pallet的[`noteImminentPreimage`](/builders/pallets-precompiles/pallets/democracy/#:~:text=noteImminentPreimage(encodedProposal)){target=_blank} 方法
+
+The interface also includes the following events:
+
+- **Proposed**(*uint32 indexed* proposalIndex, *uint256* deposit) - emitted when a motion has been proposed
+- **Seconded**(*uint32 indexed* proposalIndex, *address* seconder) - emitted when an account has seconded a proposal
+- **StandardVote**(*uint32 indexed* referendumIndex, *address* voter, *bool* aye, *uint256* voteAmount, *uint8* conviction) - emitted when an account has made a standard vote
+- **Delegated**(*address indexed* who, *address* target) - emitted when an account has delegated some voting power to another account
+- **Undelegated**(*address indexed* who) - emitted when an account has undelegated some of their voting power from another account
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/democracy.md:70]: This section is new and needs to be translated, please!

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 8]
```diff
@@ -69,6 +51,23 @@ Moonbeam的链上治理系统得益于[Substrate民主pallet](https://docs.rs/pa
...
+ - **unDelegate**() —— 为委托人用于解除投票权力的函数。在原先委托指令中的Token锁定时间参数被解除后，Token将会解锁并能够领取。调用民主pallet的[`undelegate`](/builders/pallets-precompiles/pallets/democracy/#:~:text=undelegate()){target=_blank}方法
+ - **unlock**(*address* target) —— 解锁逾期锁定的Token。在使用**unlock**之前，您必须为各个提案所锁定的Token调用**removeVote**，不然这些Token将会保持锁定。这个函数能够被任何账户调用。调用民主pallet的[`unlock`](/builders/pallets-precompiles/pallets/democracy/#:~:text=unlock(target)){target=_blank}方法
+ - **notePreimage**(*bytes* encodedProposal) —— 为即将到来的提案注册一个原像（Preimage）。此函数不需要提案处于调度队列中，但需要押金以进行操作，押金将会在提案生效后返还。调用民主pallet的[`notePreimage`](/builders/pallets-precompiles/pallets/democracy/#:~:text=notePreimage(encodedProposal)){target=_blank}方法
+ - **noteImminentPreimage**(*bytes* encodedProposal) —— 为即将到来的提案注册一个原像（Preimage）。此函数需要提案处在调度队列中，同时不需要任何押金。当调用成功，例如：原像（Preimage）尚未被上传且与近期的提案相匹配，不支付任何费用。调用民主pallet的[`noteImminentPreimage`](/builders/pallets-precompiles/pallets/democracy/#:~:text=noteImminentPreimage(encodedProposal)){target=_blank} 方法
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/democracy.md:62]: These were just moved around, no need to check these!

File: builders/pallets-precompiles/pallets/democracy.md [hunk 9]
```diff
@@ -53,28 +42,18 @@ keywords: 民主、substrate、pallet、moonbeam、波卡
...
-- **externalProposeDefault**(proposalHash) - 根据给定的提案原像哈希，且可以安排外部公投，即安排一次负面投票率的公投进行投票。在*议会超过5分之3成员*同意后，该公投将会进入下个阶段
-
-- **exernalProposeMajority**(proposal) - schedules a majority-carries referendum to be brought to a vote next once it is legal to schedule an external referendum given the preimage hash of a proposal. If at least *3/5th of the Council* vote in approval the referendum will be tabled next. Please refer to the three types of proposals, in the **propose** extrinsic description above
-- **exernalProposeMajority**(proposalHash) - 根据给定的提案原像哈希，且可以安排外部公投，即安排一次多数票的公投进行进阶投票。在*议会超过5分之3成员*同意后，该公投将会进入下个阶段
-
+- **externalPropose**(proposal) - 根据给定的提案哈希在适当计划一个外部公投时制定一个公投。在*议会超过2分之1的成员*同意后，该公投将会进入下个阶段。请参考上面的**propose** extrinsic描述中的三种类型的提案
+- **externalProposeDefault**(proposal) - 根据给定的提案原像哈希，且可以安排外部公投，即安排一次负面投票率的公投进行投票。在*议会超过5分之3成员*同意后，该公投将会进入下个阶段。请参考上面的**propose** extrinsic描述中的三种类型的提案
+- **exernalProposeMajority**(proposal) - 根据给定的提案原像哈希，且可以安排外部公投，即安排一次多数票的公投进行进阶投票。在*议会超过5分之3成员*同意后，该公投将会进入下个阶段。请参考上面的**propose** extrinsic描述中的三种类型的提案
```
Comment (eshaben) [inline builders/pallets-precompiles/pallets/democracy.md:47]:
```suggestion
- **externalProposeMajority**(proposal) - 根据给定的提案原像哈希，且可以安排外部公投，即安排一次多数票的公投进行进阶投票。在*议会超过5分之3成员*同意后，该公投将会进入下个阶段。请参考上面的**propose** extrinsic描述中的三种类型的提案
```

**Review by themacexpert — APPROVED** [review]
LGTM

**Pull request merged**